server/clusters/alerting/createAlertingCluster.js

```diff
@@ -16,9 +16,9 @@
 import alertingPlugin from './alertingPlugin';
 import { CLUSTER, DEFAULT_HEADERS } from '../../services/utils/constants';
 
-export default function createAlertingCluster(server) {
-  const { customHeaders, ...rest } = server.config().get('elasticsearch');
-  server.plugins.elasticsearch.createCluster(CLUSTER.ALERTING, {
+export default function createAlertingCluster(core, globalConfig) {
+  const { customHeaders, ...rest } = globalConfig.elasticsearch;
+  return core.elasticsearch.legacy.createClient(CLUSTER.ALERTING, {
     plugins: [alertingPlugin],
```

Comment:
You should be able to combine the alerting and AD plugins here, similar to [here](https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/blob/new-platform/server/plugin.ts#L62)